rules-tests/TypeDeclaration/Rector/Class_/ReturnTypeFromStrictTernaryRector/Fixture/short_ternary.php.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Class_\ReturnTypeFromStrictTernaryRector\Fixture;
+
+final class ShortTernary
+{
+    public function getValue()
+    {
+        $x = false;
+        if (rand(0,1)) {
+            $x = $this->get();
+        }
+        return $x ?: null;
+    }
+
+    public function get(): ShortTernary {}
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Class_\ReturnTypeFromStrictTernaryRector\Fixture;
+
+final class ShortTernary
+{
+    public function getValue(): ?\Rector\Tests\TypeDeclaration\Rector\Class_\ReturnTypeFromStrictTernaryRector\Fixture\ShortTernary
```

Comment:
not sure why this prints the FQCN.. but I guess its intended and should be fixed by other rectors or a CS tool

Reply:
that's expected, `$rectorConfig->importNames()` can cover  that